config/config.sample.php

```diff
@@ -1048,13 +1048,25 @@
  *                this condition is met
  *  - ``apps``:   if the log message is invoked by one of the specified apps,
  *                this condition is met
+ *  - ``matches``: if all the conditions inside a group match,
+ *                this condition is met. This allows to log only entries to an app
+ *                by a few users.
  *
  * Defaults to an empty array.
  */
 'log.condition' => [
 	'shared_secret' => '57b58edb6637fe3059b3595cf9c41b9',
 	'users' => ['sample-user'],
 	'apps' => ['files'],
+	'matches' => [
```

Comment:
```suggestion
	'and' => [
```
Or is that too nerdy?

Reply:
```suggestion
	'allof' => [
```
maybe?

Reply:
I had that at some point all ready, but my mind couldn't settle if it's allof or anyof and then I wondered if it's more understandable for people